ibis-server/app/model/querier.py

```diff
@@ -1,12 +1,19 @@
 from json import loads
 
 import pandas as pd
+from pydantic import BaseModel, Field
 
 from app.mdl.rewriter import rewrite
 from app.model.data_source import DataSource, ConnectionInfo
+from app.model.data_source import (
+    PostgresConnectionUrl,
+    PostgresConnectionInfo,
+    BigQueryConnectionInfo,
+    SnowflakeConnectionInfo,
+)
 
 
-class Connector:
+class Querier:
```

Comment:
I think it's not a common name, and it's actually hard to pronounce. If you don't prefer `Connector`, how about `QueryAgent`?